samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/PublishedAssessmentSettingsBean.java

```diff
@@ -1644,6 +1649,32 @@ public String getDisplayScoreDuringAssessments(){
  	public void setDisplayScoreDuringAssessments(String displayScoreDuringAssessments){
  		this.displayScoreDuringAssessments = displayScoreDuringAssessments;
  	}
+ 	
+ 	public boolean getGradebookLinkeable() {
+ 		return this.gradebookLinkeable;
+ 	}
+
+ 	public void setGradebookLinkeable(boolean pLinkeable) {
+ 		this.gradebookLinkeable = pLinkeable;
+ 	}
+
+ 	/** 
+ 	 *
+ 	 * Returns true if at the time of creating or editing an assessment, there exist an empty GB item with 
+ 	 * the same title as the assignment for it to be potentially linked to it.
+ 	 * @return
+ 	 */
+ 	private boolean gradebookItemAvailable() {
```

Comment:
isGradebookItemAvailable. It's a boolean, the getters are setters are `is` and `get`.